src/components/roverSelector/RoverSelectorPage.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+export const RoverSelectorPage: React.FunctionComponent = () => {
+    return (
+      <main>
+        hello
+      </main>
+    );
+  };
```

Comment:
See how GitHub has added a little ⛔ at the bottom of the file here? This is to indicate that the file doesn't end with a single empty line, which is usually preferred. Could you add one here please?

Reply:
Extra line added